src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -182,8 +232,27 @@ def init_mgmt_interface_tables(db_conn):
 
     return oid_name_map, if_alias_map
 
+def init_multi_sync_d_interface_tables(multi_db_conn):
+    if_name_map = {}
+    if_alias_map = {}
+    if_id_map = {}
+    oid_sai_map = {}
+    oid_name_map = {}
+
+    for db_conn in multi_db_conn:
+        if_name_map_asic, \
+        if_alias_map_asic, \
+        if_id_map_asic, \
+        oid_sai_map_asic, \
+        oid_name_map_asic = init_sync_d_interface_tables(db_conn)
+        if_name_map.update(if_name_map_asic)
+        if_alias_map.update(if_alias_map_asic)
+        if_id_map.update(if_id_map_asic)
+        oid_sai_map.update(oid_sai_map_asic)
+        oid_name_map.update(oid_name_map_asic)
+
+    return if_name_map, if_alias_map, if_id_map, oid_sai_map, oid_name_map 
 
-# TODO: the function name include interface, but only return port by design. Fix the design or the name
```

Comment:
># TODO: the function name inclu [](start = 0, length = 31)

Could you rebase your commits to latest master? I see some master changes missing in your PR.

Reply:
Rebased to master.